.circleci/config.yml

```diff
@@ -1179,14 +1179,40 @@ jobs:
           name: pip install python-dateutil
           command: python3 -m pip install python-dateutil
       - run:
-          name: pip install python-magic-bin
+          name: pip install python-magic
           command: python3 -m pip install python-magic
       - run:
           name: Run linux release
           command: |
             VERSION=${CIRCLE_TAG:1}
             python3 ./tools/release/publish_release.py --no-build-chocolatey --no-chocolatey-publish --no-build-homebrew --github-token ${GITHUB_TOKEN} --use-existing-release --version ${VERSION} --output-dir artifacts
 
+  windows_publish_release:
+    environment:
+      <<: *windows_environment
+    working_directory: "C:\\Users\\circleci\\buck"
+    executor: win/default
+    steps:
+      - checkout
+      - run:
+          name: Upgrade pip
+          command: python3 -m pip install --upgrade pip
+      - run:
+          name: pip install requests
+          command: python3 -m pip install requests
+      - run:
+          name: pip install python-dateutil
+          command: python3 -m pip install python-dateutil
+      - run:
+          name: pip install python-magic-bin
+          command: python3 -m pip install python-magic-bin
```

Comment:
These use `python-magic-bin` but the other step uses `python-magic`?

Reply:
The name of the module on Linux is python-magic and and on Mac/Windows is python-magic-bin.